forage-android/src/main/java/com/joinforage/forage/android/ForageSDK.kt

```diff
@@ -83,17 +87,15 @@ class ForageSDK : ForageSDKInterface {
     }
 
     /**
-     * Checks the balance of a given PaymentMethod using a ForagePINEditText
-     *
-     * @param params The parameters required for balance inquiries, including
-     * a reference to a ForagePINEditText and PaymentMethod ref
+     * Checks the balance of an EBT Card via a [ForagePINEditText][com.joinforage.forage.android.ui.ForagePINEditText] Element.
```

Comment:
Note for future cleanup/tickets but non-blocking: 

See comment above about generic (non-ebt-specific) comments for non-ebt-specific code^